packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -927,10 +927,7 @@ async function needMigrateToArmAndMultiEnv(ctx: CoreHookContext): Promise<boolea
 }
 
 function preCheckEnvEnabled() {
-  if (isArmSupportEnabled()) {
-    return true;
-  }
-  return false;
+  return true;
```

Comment:
we can remove this function directly and update code that calls this function

Reply:
Will fix in next PR.